.env.example

```diff
@@ -2,3 +2,5 @@ REACT_APP_API_PREFIX="api"
 REACT_APP_GLIFIC_API_PORT=4000
 REACT_APP_APPSIGNAL_API_KEY=""
 REACT_APP_APPLICATION_NAME="Glific: Two way communication platform"
+LOGFLARE_API_KEY = 'vV5sj8-sZZIa'
```

Comment:
Please remove from API keys from here

Reply:
@DigneshGujarathi Can you also please change the API key in logflare.